Move some changeset to the "Removals" section of CHANGELOG (#4290)
OpenZeppelin/openzeppelin-contracts: CHANGELOG.md

### Removals

The following contracts, libraries and functions were removed:

- `Address.isContract` (because of its ambiguous nature and potential for misuse)
- `Checkpoint.History`
- `ERC20Snapshot`
- `ERC20VotesComp`
- `ERC165Storage` (in favor of inheritance based approach)
- `ERC777`
- `ERC1820Implementer`
- `GovernorVotesComp`
- `GovernorProposalThreshold` (deprecated since 4.4)
- `PaymentSplitter`
- `PullPayment`
- `SafeMath`
- `SignedSafeMath`
- `Timers`
- `TokenTimelock` (in favor of `VestingWallet`)
- All escrow contracts (`Escrow`, `ConditionalEscrow`, `RefundEscrow`)

> - All crossChain contracts, including `AccessControlCrossChain` and all the vendored bridge interfaces

Reviewer: ```suggestion
- All CrossChain contracts, including `AccessControlCrossChain` and all the vendored bridge interfaces
```

> - All CrossChain contracts, including `AccessControlCrossChain` and all the vendored bridge interfaces

Author: Should we do lowercase "crosschain" following the folder's name?
```suggestion
- All crosschain contracts, including `AccessControlCrossChain` and all the vendored bridge interfaces
```

Reviewer 2: ```suggestion
- All cross-chain contracts, including `AccessControlCrossChain` and all the vendored bridge interfaces
```

### Removals

The following contracts, libraries and functions were removed:

- `Address.isContract` (because of its ambiguous nature and potential for misuse)

> - `Checkpoint.History`

Reviewer: ```suggestion
- `Checkpoints.History`
```

- `ERC20Snapshot`
- `ERC20VotesComp`
- `ERC165Storage` (in favor of inheritance based approach)
- `ERC777`
- `ERC1820Implementer`
- `GovernorVotesComp`
- `GovernorProposalThreshold` (deprecated since 4.4)
- `PaymentSplitter`
- `PullPayment`
- `SafeMath`
- `SignedSafeMath`
- `Timers`
- `TokenTimelock` (in favor of `VestingWallet`)

> - All escrow contracts (`Escrow`, `ConditionalEscrow`, `RefundEscrow`)

Reviewer: ```suggestion
- All escrow contracts (`Escrow`, `ConditionalEscrow` and `RefundEscrow`)
```
